### PR TITLE
Fix Ember.Logger deprecation

### LINKED ISSUE #134: Fix `Ember.Logger` deprecation
https://deprecations-app-prod.herokuapp.com/deprecations/v3.x/#toc_ember-console-deprecate-logger

### PR DESCRIPTION
**Pull request by mydea**
Replace Ember.Logger with console to fix deprecation warnings.

Fixes #134

### CODE REVIEW

File: addon/services/raven.js [hunk 1]
```diff
@@ -50,15 +48,19 @@ export default Service.extend({
    * @property ignoreErrors
    * @type Array
    */
-  ignoreErrors: [],
+  ignoreErrors: computed(function() {
+    return [];
+  }),
```
Comment (Turbo87) [inline addon/services/raven.js]: @mydea can you remove these unrelated changes from the PR?

File: addon/index.js [hunk 2]
```diff
@@ -10,7 +9,7 @@ export function initialize(container, config) {
 
   if (get(config, 'sentry.development') === true) {
     if (get(config, 'sentry.debug') === true) {
-      Ember.Logger.info('`sentry` is configured for development mode.');
+      console.info('`sentry` is configured for development mode.'); // eslint-disable-line no-console
```
Comment (Turbo87) [inline addon/index.js]: I guess we should probably disable the `no-console` rule globally instead of overriding it for every line where we use it 🤔

**Comment by dashdashzako** [comment]
Changes were addressed in #141, closing.
